### DISCUSSION TITLE
ldaps cannot connect server

### DISCUSSION THREAD

**Discussion by conan522** [snipe/snipe-it#17571] [Q&A]
### Debug mode

- [X] I have enabled debug mode
- [X] I have read [checked the Common Issues page](https://snipe-it.readme.io/docs/common-issues)

### Describe the bug

ldaps cannot connect server. but ldap works

### Reproduction steps

1.ldap:\\domain.com\ works
2.ldaps:\\domain.com:636\ does not work
3.
...

### Expected behavior

ldaps:\\domain.com:636\ work

### Snipe-IT Version

v4.9.4

### Operating System

Windwos Server 2016

### Web Server

Apche

### PHP Version

7.0

### Operating System

Windows 10

### Browser

Google Chrome

### Version

93

### Error messages

```shell
Could not bind to LDAP: Can't contact LDAP server
```

**Comment by conan522** [comment]
I change the LDAP server from ldap://domain.com/ to ldaps://domain.com:636/, and check the "Allow invalid SSL Certificate". Is it right? Also I do not know where I can import the pem CA or fill in CA's path. Anyone can guide me, thanks a lot.

**Comment by conan522** [comment]
Anyone have a experience for ldaps configuration?

**Comment by snipe** [comment]
Hi there - I know this is a pretty old issue, but I wanted to check and see if you ever got this resolved? If so, can you please close the ticket? If not, can you update us on what version of Snipe-IT you're currently using, so we can see if it's still an issue on latest?

:) Thanks!
